Docs/Status.md

```diff
@@ -7,151 +7,301 @@ Foundation is divided into groups of related functionality. These groups are cur
+    | `NSMutableSet`        | Mostly Complete | ?             | `init?(coder:)` remains unimplemented                                                                                                                               |
+    | `NSCountedSet`        | Mostly Complete | ?             | `init?(coder:)` remains unimplemented                                                                                                                               |
+    | `NSCFSet`             | N/A             | ?             | For internal use only                                                                                                                                               |
+    | `NSCache`             | Complete        | ?             |                                                                                                                                                                     |
+    | `NSSortDescriptor`    | Unimplemented   | ?             |                                                                                                                                                                     |
 
 * **RunLoop**: Timers, streams and run loops.
 
     The classes in this group provide support for scheduling work and acting upon input from external sources.
 
-    * `NSPort` is not yet implemented.
-    * `NSPortMessage` is not yet implemented.
-    * `NSRunLoop` is not yet implemented. Its CoreFoundation equivalent `CFRunLoop` is available, but needs some work to remove a dependency on libdispatch.
-    * `NSStream` is not yet implemented.
-    * `NSTimer` is not yet implemented.
-
+    | Entity Name      | Status          | Test Coverage | Notes                                                                         |
+    |------------------|-----------------|---------------|-------------------------------------------------------------------------------|
+    | `Port`           | Unimplemented   | ?             |                                                                               |
+    | `MessagePort`    | Unimplemented   | ?             |                                                                               |
+    | `SocketPort`     | Unimplemented   | ?             |                                                                               |
+    | `PortMessage`    | Unimplemented   | ?             |                                                                               |
+    | `RunLoop`        | Mostly Complete | ?             | `add(_: Port, forMode:)` and `remove(_: Port, forMode:)` remain unimplemented |
+    | `NSStream`       | Mostly Complete | ?             |                                                                               |
+    | `Stream`         | Unimplemented   | ?             | Methods which do not require a concrete implementation remain unimplemented   |
```

Comment:
Good eye, thanks for catching this. I'd meant the first one — all methods which are not implemented in terms of `NSRequiresConcreteImplementation()` have been left `NSUnimplemented()`, and need implementations. I will push a correction.